<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pixyl Uploader</string>
  </property>
  <property name="windowIcon">
   <iconset resource="graphics.qrc">
    <normaloff>:/camera.ico</normaloff>:/camera.ico</iconset>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_1">
       <attribute name="title">
        <string>Queue</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QToolButton" name="addQueueButton">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="graphics.qrc">
              <normaloff>:/icon/add_photo</normaloff>:/icon/add_photo</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="removeQueueButton">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="graphics.qrc">
              <normaloff>:/icon/delete</normaloff>:/icon/delete</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="clearQueueButton">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="graphics.qrc">
              <normaloff>:/icon/clear_all</normaloff>:/icon/clear_all</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="queueTableView">
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="textElideMode">
           <enum>Qt::ElideLeft</enum>
          </property>
          <property name="verticalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
          <property name="horizontalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Watch Folders</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QToolButton" name="addFolderButton">
            <property name="toolTip">
             <string>Add folder to watchlist</string>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="graphics.qrc">
              <normaloff>:/icon/add</normaloff>:/icon/add</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="removeFolderButton">
            <property name="toolTip">
             <string>Remove folder from watchlist</string>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="graphics.qrc">
              <normaloff>:/icon/remove</normaloff>:/icon/remove</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="clearWatchlistButton">
            <property name="toolTip">
             <string>Clear watchlist</string>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="graphics.qrc">
              <normaloff>:/icon/clear_all</normaloff>:/icon/clear_all</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="watchTableView">
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="textElideMode">
           <enum>Qt::ElideLeft</enum>
          </property>
          <property name="verticalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
          <property name="horizontalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>750</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionResume"/>
   <addaction name="actionPause"/>
   <addaction name="actionStop"/>
   <addaction name="separator"/>
   <addaction name="actionCreateAlbum"/>
   <addaction name="separator"/>
   <addaction name="actionLogIn"/>
   <addaction name="actionLogOut"/>
   <addaction name="separator"/>
   <addaction name="actionSetting"/>
   <addaction name="actionEmail"/>
   <addaction name="actionSMS"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionResume">
   <property name="icon">
    <iconset resource="graphics.qrc">
     <normaloff>:/icon/resume</normaloff>:/icon/resume</iconset>
   </property>
   <property name="text">
    <string>Resume</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="graphics.qrc">
     <normaloff>:/icon/stop</normaloff>:/icon/stop</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionSetting">
   <property name="icon">
    <iconset resource="graphics.qrc">
     <normaloff>:/icon/setting</normaloff>:/icon/setting</iconset>
   </property>
   <property name="text">
    <string>Setting</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="icon">
    <iconset resource="graphics.qrc">
     <normaloff>:/icon/pause</normaloff>:/icon/pause</iconset>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
  </action>
  <action name="actionCreateAlbum">
   <property name="icon">
    <iconset resource="graphics.qrc">
     <normaloff>:/icon/photo_album</normaloff>:/icon/photo_album</iconset>
   </property>
   <property name="text">
    <string>CreateAlbum</string>
   </property>
   <property name="toolTip">
    <string>Create Album</string>
   </property>
  </action>
  <action name="actionLogIn">
   <property name="icon">
    <iconset resource="graphics.qrc">
     <normaloff>:/icon/log_in</normaloff>:/icon/log_in</iconset>
   </property>
   <property name="text">
    <string>LogIn</string>
   </property>
   <property name="toolTip">
    <string>Log In</string>
   </property>
  </action>
  <action name="actionLogOut">
   <property name="icon">
    <iconset resource="graphics.qrc">
     <normaloff>:/icon/log_out</normaloff>:/icon/log_out</iconset>
   </property>
   <property name="text">
    <string>LogOut</string>
   </property>
   <property name="toolTip">
    <string>Log Out</string>
   </property>
  </action>
  <action name="actionEmail">
   <property name="icon">
    <iconset resource="graphics.qrc">
     <normaloff>:/icon/email</normaloff>:/icon/email</iconset>
   </property>
   <property name="text">
    <string>Email</string>
   </property>
   <property name="toolTip">
    <string>Email</string>
   </property>
  </action>
  <action name="actionSMS">
   <property name="icon">
    <iconset resource="graphics.qrc">
     <normaloff>:/icon/sms</normaloff>:/icon/sms</iconset>
   </property>
   <property name="text">
    <string>SMS</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="graphics.qrc"/>
 </resources>
 <connections/>
</ui>
